tests/integration/test_feedback.py

```diff
@@ -42,24 +43,33 @@ def generate_feedback_sdk_event():
     }
 
 
+@pytest.mark.parametrize("use_feedback_topic", (False, True))
 def test_feedback_event_with_processing(
-    mini_sentry, relay_with_processing, events_consumer
+    mini_sentry,
+    relay_with_processing,
+    events_consumer,
+    feedback_consumer,
+    use_feedback_topic,
 ):
-    relay = relay_with_processing()
     mini_sentry.add_basic_project_config(
         42, extra={"config": {"features": ["organizations:user-feedback-ingest"]}}
     )
 
-    _events_consumer = events_consumer(timeout=20)
-    feedback = generate_feedback_sdk_event()
+    if use_feedback_topic:
+        mini_sentry.set_global_config_option("feedback.ingest-topic.rollout-rate", 1.0)
+        consumer = feedback_consumer(timeout=20)
+    else:
+        mini_sentry.set_global_config_option("feedback.ingest-topic.rollout-rate", 0.0)
+        consumer = events_consumer(timeout=20)
```

Comment:
Might be worth also assigning `other_consumer` and calling `.assert_empty()` on it at the end of the test.